### PR TITLE
Fixes for trivial PSR violations

### PR DESCRIPTION
**Pull request by vmcj**
No functional changes intended

### CODE REVIEW

File: webapp/config/lib_load_db_secrets.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
```
Comment (nickygerritsen) [inline webapp/config/lib_load_db_secrets.php:1]: Why is this put in a separate file?

**Comment by vmcj** [comment]
I think https://github.com/DOMjudge/domjudge/pull/1706/commits/a349a34962bebc22217e1f102df77e8c1814746c is the one which might need the most discussion. I don't think we ever change the file and this is one of the corner cases where the original file also made sense.

File: webapp/config/lib_load_db_secrets.php (hunk 1 above)
Comment (vmcj) [reply webapp/config/lib_load_db_secrets.php:1]: Reasoning is this:
```
--------------------------------------------------------------------------------------------------------FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE                                                           -------------------------------------------------------------------------------------------------------- 1 | WARNING | A file should declare new symbols (classes, functions, constants, etc.) and cause no        |         | other side effects, or it should execute logic with side effects, but should not do         |         | both. The first symbol is defined on line 9 and the first side effect is on line 51.     --------------------------------------------------------------------------------------------------------                                                                                                        Time: 56ms; Memory: 6MB
```

**Comment by nickygerritsen** [comment]
> I think [a349a34](https://github.com/DOMjudge/domjudge/commit/a349a34962bebc22217e1f102df77e8c1814746c) is the one which might need the most discussion. I don't think we ever change the file and this is one of the corner cases where the original file also made sense.

What was the error we were getting? Would it make sense to ignore this file for checks? I'm not really happy with splitting it just to make CI happy.

**Comment by vmcj** [comment]
> > I think [a349a34](https://github.com/DOMjudge/domjudge/commit/a349a34962bebc22217e1f102df77e8c1814746c) is the one which might need the most discussion. I don't think we ever change the file and this is one of the corner cases where the original file also made sense.
> 
> What was the error we were getting? Would it make sense to ignore this file for checks? I'm not really happy with splitting it just to make CI happy.

I've posted it inline in your original comment,

And we could ignore the file,
Or could remove the PSR1 rule, 
Or accept the annotation if someone ever changes something in that file as we only annotate the changed files.

**Comment by nickygerritsen** [comment]
> > > I think [a349a34](https://github.com/DOMjudge/domjudge/commit/a349a34962bebc22217e1f102df77e8c1814746c) is the one which might need the most discussion. I don't think we ever change the file and this is one of the corner cases where the original file also made sense.
> > 
> > 
> > What was the error we were getting? Would it make sense to ignore this file for checks? I'm not really happy with splitting it just to make CI happy.
> 
> I've posted it inline in your original comment,
> 
> And we could ignore the file, Or could remove the PSR1 rule, Or accept the annotation if someone ever changes something in that file as we only annotate the changed files.

I would vote for any of these, since I don't think this rule will apply to any other file.

**Comment by vmcj** [comment]
> > > > I think [a349a34](https://github.com/DOMjudge/domjudge/commit/a349a34962bebc22217e1f102df77e8c1814746c) is the one which might need the most discussion. I don't think we ever change the file and this is one of the corner cases where the original file also made sense.
> > > 
> > > 
> > > What was the error we were getting? Would it make sense to ignore this file for checks? I'm not really happy with splitting it just to make CI happy.
> > 
> > 
> > I've posted it inline in your original comment,
> > And we could ignore the file, Or could remove the PSR1 rule, Or accept the annotation if someone ever changes something in that file as we only annotate the changed files.
> 
> I would vote for any of these, since I don't think this rule will apply to any other file.

I think I agree, thats why I already explicitly mentioned that one. Will take it out to see if that also makes CI happy

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**